actors/migration/nv10/util.go

```diff
@@ -130,21 +130,19 @@ func migrateHAMTAMTRaw(ctx context.Context, store cbor.IpldStore, root cid.Cid,
 }
 
 type MemMigrationCache struct {
-	MigrationMap *sync.Map
+	MigrationMap sync.Map
```

Comment:
Now that we're returning `MemMigrationCache` by pointer, we don't need to put the map behind a pointer.